cpp/src/io/copc_reader.cpp

```diff
@@ -16,67 +16,20 @@
 namespace copc
```

Comment:
There are a couple unused includes

Reply:
I can't detect them in my IDE unfortunately

Reply:
Those two
```
#include "copc-lib/las/header.hpp"
#include <lazperf/readers.hpp>
```